presto-verifier/src/main/java/com/facebook/presto/verifier/framework/DataVerificationConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Curious why we are extracting this out to a separate config? Do we envision more such properties in the future?

Reply:
I was thinking that we're only support `skip-control` for DML verification, so put this as a separate config that are specific to DML verification. But you reminded me that we could also trivially support this config for explain as well.
Removed this config and add support for `skip-control` for explain in the last commit.